aprovado_ou_reprovado/spec/aprovado_ou_reprovado_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_base'
+require 'aprovado_ou_reprovado'
+
+describe 'Aprovado ou reprovado' do
+
+  it 'deve calcular a média de 2 números sendo n1 negativo e retornar nota inválida' do
```

Comment:
Acho que aqui a gente pode agrupar alguns cenários para não precisar repetir os contextos, o que acha?  Poderia ficar assim, por exemplo: 
```suggestion
  context 'deve calcular a média de 2 números e retornar nota inválida' do
    it 'quando n1 possui valor negativo' do
      expect(media(-0.1,0)).to eq('nota inválida')
    end
    it 'quando n2 possui valor negativo' do
      expect(media(0,-0.1)).to eq('nota inválida')
    end
    it 'quando n1 possui valor maior que 10' do
      expect(media(10.1,10)).to eq('nota inválida')
    end
     it 'quando n2 possui valor maior que 10' do
       expect(media(10,10.1)).to eq('nota inválida')
    end
  end
```